sdk/appconfiguration/Azure.ApplicationModel.Configuration/src/ConfigurationClientOptions.cs

```diff
@@ -9,17 +9,13 @@ namespace Azure.ApplicationModel.Configuration
 {
     public class ConfigurationClientOptions: HttpClientOptions
     {
+        public static string AuthenticationPolicy { get; } = "Authentication";
+        public static string BufferResponsePolicy { get; } = "BufferResponse";
```

Comment:
Let;s not add public APIs to the mainline scenario APIs for very corner case scenarios.

Reply:
Done